src/main/typescript/constants/serverRoutes.ts

```diff
@@ -21,6 +21,7 @@ const createURL: (path: string) => Promise<string> = async path => {
     return url.endsWith("/") ? `${url}${path}` : `${url}/${path}`
 }
 
+export const listFilesURL: (id: DepositId, dirPath: string) => Promise<string> = (id, dirPath) =>createURL(`deposit/${id}/file/${dirPath}`)
```

Comment:
Related to https://github.com/DANS-KNAW/easy-deposit-ui/pull/69/review_comment/create#r209164734, are you aware that this URL already exists below at

https://github.com/DANS-KNAW/easy-deposit-ui/blob/a987c4b62b42d523bd725277c328f085fcf654f3/src/main/typescript/constants/serverRoutes.ts#L40

However, there it is without the `dirPath` argument.